tests/framework/scheduler.py

```diff
@@ -89,6 +91,26 @@ def pytest_runtest_logreport(self, report):
         """
         self._add_report(report)
 
+    @staticmethod
+    def deselect_if_marked(config, batch, marker_name):
```

Comment:
I would rename this to something like `filter_batch()`

Reply:
Done.